Visual/test/test_bff_demo.py

```
@@ -27,13 +27,17 @@ def tearDownClass(cls):
 
   def test_01_expand_collapse_nodes(self):
     global driver
+    time.sleep(5)
     nodes = driver.find_elements_by_class_name('node')
     # Page opens with some nodes expanded
     oldSize = len(nodes)
-    # Click on root node to collapse all nodes
-    nodes[-1].find_element_by_tag_name("path").click()
-    time.sleep(1)
+    # Click on root node to collapse all nodes"
+    try:
+        nodes[-1].find_element_by_tag_name("path").click()
+    except:
+        nodes[-1].find_element_by_tag_name("path").click()
```

Why is your "recovery" from an error just to do the same thing again?
Won't this just generate the same error you just got?
If it does, then why not just propagate the original error instead of creating a new one
that is exactly alike?

Inquiring minds want to know.. (GRIN)